connectors/sources/mssql.py

```diff
@@ -79,44 +79,53 @@ def get_default_configuration(cls):
                     "type": "str",
                     "value": "dbo",
                 },
-                "mssql_driver": {
-                    "label": "Microsoft SQL Driver Name (ODBC Driver 18 for SQL Server)",
+                "ssl_enabled": {
+                    "display": "toggle",
+                    "label": "Enable SSL verification (true/false)",
                     "order": 10,
-                    "type": "str",
-                    "value": "ODBC Driver 18 for SQL Server",
+                    "type": "bool",
+                    "value": False,
                 },
-                "secured_connection": {
-                    "display": "toggle",
-                    "label": "Connection will be secured or not",
+                "ssl_ca": {
+                    "label": "Certificate Data",
                     "order": 11,
+                    "type": "str",
+                    "value": "",
+                },
+                "validate_host": {
+                    "label": "Do you want to validate host",
+                    "order": 12,
                     "type": "bool",
-                    "value": SECURED_CONNECTION,
+                    "value": False,
                 },
             }
         )
         return mssql_configuration
 
     def _create_engine(self):
         """Create sync engine for mssql"""
-        if self.secured_connection:
-            query = {
-                "driver": self.mssql_driver,
-                "TrustServerCertificate": "no",
-                "Encrypt": "Yes",
-            }
-        else:
-            query = {"driver": self.mssql_driver, "TrustServerCertificate": "yes"}
-
         connection_string = URL.create(
-            "mssql+pyodbc",
+            "mssql+pytds",
             username=self.user,
             password=self.password,
             host=self.host,
             port=self.port,
             database=self.database,
-            query=query,
         )
-        self.engine = create_engine(connection_string)
+        connect_args = {}
+        if self.ssl_enabled:
+            self.create_pem_file()
+            connect_args = {
+                "cafile": "ssl_certificate_mssql.pem",
+                "validate_host": self.validate_host,
+            }
+        self.engine = create_engine(connection_string, connect_args=connect_args)
+
+    def create_pem_file(self):
+        """Create pem file for SSL Verification"""
+        pem_format = get_pem_format(key=self.ssl_ca, max_split=1)
+        with open("ssl_certificate_mssql.pem", "w") as cert:
```

Comment:
Should you create somewhere in `/tmp`?

Also what if you have several MSSQL connectors running together?

Reply:
Sure, we will move this file to the`/tmp` folder. Also to avoid conflict we append the current time in the file name.

Reply:
done, moved to `/tmp`